resources/default_pipeline_environment.yml

```diff
@@ -109,6 +109,9 @@ steps:
     stashContent:
       - 'tests'
     testPackage: 'piper-bats'
+  checkmarxExecuteScan:
+    clientId: resource_owner_client
+    clientSecret: 014DF517-39D1-4453-B7B3-9930C563627C
```

Comment:
Why also here when it is already part of the YAML?

Reply:
Valid finding, thanks. Just removed it.